src/scripts/main.coffee

```diff
@@ -119,7 +146,21 @@ class FormbuilderCollection extends Backbone.Collection
       correctType = model.get('type') in ['dropdown', 'checkbox', 'radio', 'approval']
       differentModel = model != child
       hasNoParent = !model.hasParent()
-      correctType and differentModel and hasNoParent
+      flag = true
+      uuid = child.attributes.uuid
+      uuid_parent
+      ascent
+      parent = model.conditionalParent()
+      if parent
+        ascent = model.findConditionalParent()
```

Comment:
I know this is picky, but just to make it clear to another developer, given the output of the fincConditionalParent() is an array, is it possible to name the `ascent` variable something like `ancestorUuids`?